scripts/log-viewer.sh

```
@@ -0,0 +1,97 @@
+    esac
+done
+
+if [ ! -f "$LOG_FILE" ]; then
+    echo "No log file found: $LOG_FILE"
+    echo "Set LLAMA_LOG_FILE or ensure events are being logged."
+    exit 1
+fi
+
+# Format a single JSON line to human-readable
+format_line() {
+    local line="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)